app.js

```diff
@@ -12,27 +12,30 @@ var { CONFIG } = require("./helpers/config");
 // DB connection
 var MONGODB_URL = CONFIG.MONGODB_URL;
 var mongoose = require("mongoose");
-mongoose.connect(MONGODB_URL, { useNewUrlParser: true, useUnifiedTopology: true }).then(() => {
-	if (process.env.NODE_ENV !== "test") {
-		console.log("Connected to %s", MONGODB_URL);
-		console.log("App is running ... \n");
-		console.log("Press CTRL + C to stop the process. \n");
-	}
-})
-	.catch(err => {
-		console.error("App starting error:", err.message);
-		process.exit(1);
-	});
+mongoose
+  .connect(MONGODB_URL, { useNewUrlParser: true, useUnifiedTopology: true })
+  .then(() => {
+    if (process.env.NODE_ENV !== "test") {
+      console.log("Connected to %s", MONGODB_URL);
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 6 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
			console.log("Connected to %s", MONGODB_URL);
```